chart/scss/components/area.scss

```diff
@@ -0,0 +1,3 @@
+.md-chart-area {
+  fill: #00a0d1;
+}
```

Comment:
> Wow . this is a lot of work. Good job. I am concerned about all the hex values for colors, it would be so much easier to maintain if we could use variables for those colors.

Thanks for reviewing into it.  I will update the css part later when most of the feature is ready.

Reply:
Resolved in the latest PR.